cmd/porter/bundle.go

```diff
@@ -102,6 +102,7 @@ Porter uses the Docker driver as the default runtime for executing a bundle's in
 For example, the 'debug' driver may be specified, which simply logs the info given to it and then exits.`,
 		Example: `  porter bundle install
   porter bundle install MyAppInDev --file myapp/bundle.json
+  porter bundle install --parameter-set azure --param test-mode=true --param header-color=blue
   porter bundle install --param-file base-values.txt --param-file dev-values.txt --param test-mode=true --param header-color=blue
```

Comment:
I was wonderin' in the `Notes for the reviewer` -- what do you think/prefer?  I can go either way (yes, it would happen in an immediate follow-up if we prefer).

Reply:
oops! Sorry I dove straight in and missed that! 😊 Yeah I think this would work well as a follow-up. 👍